interview-questions.md

```diff
@@ -0,0 +1,58 @@
+# ASSESSMENT 3: Interview Practice Questions
+
+Answer the following questions.
+
+First, without external resources. Challenge yourself to answer from memory as if you were in a job interview.
+
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+1. What is JSX? What is one syntax difference between HTML and JSX?
+
+Your answer: JSX is a way to write HTML-like code in JavaScript and is commonly used in React. One syntax difference between HTML and JSX is the casing. onclick is HTML and onClick is JSX
+
+Researched answer: JSX is a syntax extension for JavaScript that allows you to write HTML-like code within your JavaScript code, primarily used in React applications to describe the structure and content of components. It lets you combine HTML and JavaScript into a single file, making your code more modular and easier to manage. The main syntax difference between HTML and JSX is the way attributes are defined. In HTML, attributes are defined using the regular HTML syntax, where you have the attribute name followed by an equals sign, then the value in quotes.
```

Comment:
Awesome, I love where you were going with this answer and talking about attributes. How are they done in JSX?